OneBusAway/credits.html

```diff
@@ -366,6 +366,7 @@ <h2 id='ptezdashboard'>Lumberjack Console</h2>
       <li><a href='https://thenounproject.com/term/update/698616'>Update</a> - created by Ricardo Cardoso</li>
       <li><a href='https://thenounproject.com/term/warning/33183/'>Warning</a> - created by Milky - Digital innovation, FR</li>
     </ul>
+    <p>This project also includes Material Icons by Google Inc. licensed under the terms of Apache 2.0</p>
```

Comment:
It looks like they [made the change](https://github.com/google/material-design-icons/commit/68e015dbbb6b730b5fe4934e8507cd5a465c8a3d) fairly recently